contracts/TestCampaign.sol

```diff
@@ -124,20 +172,79 @@ contract TestCampaign {
 
         let CampaignTestContract = new web3.eth.Contract(contractABI, campaignAddr);
 
-        CampaignTestContract.getPastEvents("donationMade", options, (error, events) => {
+        CampaignTestContract.getPastEvents("DonationMade", options, (error, events) => {
         if (error) {
             console.log(error);
         } else {
             //for each event, parse out sender and value
             //sender.transfer(value)
-            //emit hasRefunded(sender, value);
+            //emit HasRefunded(sender, value);
         }
         */
     }
 
+    /**
+     * @notice Allows the CampaignFactory contract to transfer any unclaimed donations to itself in the
+     * event that campaign or managing beneficiary is deemed to be untrustworthy
+     * @dev The time to call this function is left to the discretion of the CampaignFactory contract
+     */
     function returnRemainingBalance() public campaignFactoryOnly distrustOnly {
         uint256 remainingBalance =  address(this).balance;
         campaignFactory.transfer(remainingBalance);
-        emit hasReturnedBalance(address(this), remainingBalance);
+        emit HasReturnedBalance(address(this), remainingBalance);
+    }
+
+    /**
```

Comment:
Likewise

Reply:
Most of the documentation have been removed in f37357b as per your subsequent suggestion on `TestCampaignFactory.sol`